roles/openshift_metering/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+openshift_metering_operator_image: ''
+
+openshift_metering_namespace: openshift-metering
```

Comment:
I think the general practice now is to be opinionated about this: exposing the namespace config is unnecessary (and even potentially harmful at this point)

Reply:
It's useful for testing since we can't install multiple copies into a single namespace yet. I want to be able to run this playbook against my cluster for testing so I'd like to be able to parameterize this.

Reply:
No objection to making things configurable to facilitate development, but I would at least comment it with a "DO NOT CHANGE THIS" sort of thing or otherwise find a way to bury it. Any tips for this, @smarterclayton?

Reply:
I could add another boolean var as an additional check, like `openshift_metering_custom_namespace_enabled: false # ENABLE AT OWN RISK`

Reply:
If you want to bury it, I'd recommend putting it into `vars/main.yml` instead

Reply:
Reading about it and it seems like this is probably a good improvement regardless. I made the change to moving it into `vars`. Is this sufficient?